docs/android/overview/_shared/_MealzStoreSetupProvider.mdx

```diff
@@ -1,8 +1,30 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-```kotlin
-// From anywhere
-import com.miam.core.Mealz
+<Tabs
+    defaultValue="4.0"
+    groupId="mobile-version"
+    values={[
+        { label: '4.0', value: '4.0' },
+        { label: '4.1', value: '4.1' },
+    ]}>
```

Comment:
Don't use a tab old method still exist we can put a deprecated on them and add  new method